code/modules/mob/living/carbon/hallucination.dm

```diff
@@ -0,0 +1,1102 @@
+						"[holder] disappoints us all once again.")
+			sound_to(holder, 'sound/misc/announcements/notice.ogg')
+			to_chat(holder, "<h2 class='alert'>Station Announcement</h2>")
+			to_chat(holder, SPAN_ALERT(pick(body)))
+			to_chat(holder, SPAN_ALERT("-[LAZYPICK(hal_sender, holder)]"))
+
+/datum/hallucination/announcement/proc/delam_call()	//for REALLY selling that fake delamination
+	var/list/people = list()
+	for(var/mob/living/carbon/human/M in living_mob_list)
+		if(!M.isMonkey() && !player_is_antag(M, only_offstation_roles = TRUE))		//Antag check prevents meta
+			people += M
+	people -= holder
+	if(!people.len)
+		return
+	var/radio_exclaim = pick("Oh SHIT!", "Oh fuck.", "Uhhh!", "That's not good!", "FUCK.", "Engineering?", "It's under control!", "We're fucked!", "Ohhhh boy.", "What?!", "Um, <b>what?!</b>")
+	to_chat(holder, "<font color='#008000'><b>[pick(people)]</b> says, \"[radio_exclaim]\"</font>")
+
+
+
+/datum/hallucination/pda				//fake PDA messages. this only plays the beep and sends something to chat; it won't show up in the PDA.
+	min_power = 20
+	duration = 900					//this duration length + not allowing duplicates prevents spamming messages on every valid handle_hallucination() which can get VERY annoying if rng decides to give you 3 in a row
+	allow_duplicates = FALSE
+
+/datum/hallucination/pda/start()
+	var/list/sender = message_sender
+	var/mob/living/carbon/human/M = holder
+	for(var/mob/living/carbon/human/H in living_mob_list)
+		if(H.client && !player_is_antag(H, only_offstation_roles = TRUE))	//adds current players to default list to provide variety. leaves out offstation antags.
+			sender += H
+	holder.show_message("<b>Message from [pick(sender)] to [holder.name] ([M.job]),</b> \"[pick(hallucinated_phrases)]\" (<FONT color = blue><u>reply</u></FONT>)")
+	sound_to(holder, 'sound/machines/twobeep.ogg')
+
+
+/datum/hallucination/paranoia
+
+/datum/hallucination/paranoia/can_affect(mob/living/carbon/C)
+	if(!..())
+		return FALSE
+	for(var/mob/living/M in oview(C))
+		return TRUE
+
+/datum/hallucination/paranoia/start()		//hallucinate someone else doing something. Yes, it's intentional that it's any living mob, not just other characters.
+	var/list/hal_target = list() //The mob you're going to imagine doing this
+	var/firstname = copytext(holder.real_name, 1, findtext(holder.real_name, " "))
+	var/t = pick(hallucinated_actions)
+	t = replace_characters(t, list("you" = "[firstname]"))		//the list contains items that say "you." This replaces "you" with the hallucinator's first name to sell the fact that the person is doing the emote.
+	for(var/mob/living/M in oview(holder))
+		hal_target += M
+	if(hal_target.len)
+		to_chat(holder, "<b>[pick(hal_target)]</b> [t]")
+
+/datum/hallucination/paranoia/second		//Just so we get another chance at picking this.
+
+
+
+
+/datum/hallucination/skitter
```

Comment:
All this whitespace needs to be removed. It's all over this file.

Reply:
Was done for readability. Will adjust